includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -718,7 +718,7 @@ public function payment_fields() {
 	public function process_payment( $order_id ) {
 
 		if ( defined( 'WCPAY_NEW_PROCESS' ) && true === WCPAY_NEW_PROCESS ) {
-			$new_process = new PaymentProcessingService();
+			$new_process = wcpay_get_container()->get( PaymentProcessingService::class );
```

Comment:
Note: The service should be loaded within the constructor of the class, and used later. However, that would require fixing a ton of tests ([ref](https://github.com/Automattic/woocommerce-payments/pull/6421/files)), and this PR already has too many changes.